app/views/service_areas/_form.html.erb

```diff
@@ -2,13 +2,17 @@
   <%= f.error_notification %>
 
   <div class="form-inputs">
-    <%= f.input :name %>
-    <%= f.association :parent %>
-    <%= f.input :service_area_type %>
+    <div class="field is-grouped">
+      <%= f.input :name, as: :text, input_html: { rows: 1, style: "height: 2.4em;" } %>
+      <%= f.input :service_area_type, as: :select, collection: @service_area_types %>
+      <%= f.association :parent %>
+      <%= f.association :organization %>
+    </div>
     <%= f.input :description, as: :text, input_html: { rows: 2 } %>
 
-    <%= f.association :organization %>
-
+    <%= f.simple_fields_for :location, as: :location do |ff| %>
+      <%= render "location_fields", f: ff %>
+    <% end %>
```

Comment:
Added nested Location to ServiceArea. Didn't need to add link_to_add_fields bc all ServiceAreas now have (one?) Location.